src/main/java/net/fabricmc/accesswidener/AccessWidenerVisitor.java

```diff
@@ -31,6 +31,10 @@
 		this.accessWidener = accessWidener;
 	}
 
+	public static ClassVisitor createClassVisitor(int api, ClassVisitor visitor, AccessWidener accessWidener) {
```

Comment:
You need to make the visitor class public because this has been moved to here.

Reply:
Ah yeah, Might almost be worth exposing this stuff out via an api interface, but this will do for now.